pallas-applying/src/utils.rs

```diff
@@ -659,15 +639,27 @@ pub fn is_byron_address(address: &[u8]) -> bool {
 }
 
 pub fn aux_data_from_alonzo_minted_tx<'a>(mtx: &'a AlonzoMintedTx) -> Option<&'a [u8]> {
-    mtx.auxiliary_data.as_ref().map(|x| x.raw_cbor()).into()
+    if let Some(_) = mtx.auxiliary_data.deref() {
+        Some(mtx.auxiliary_data.raw_cbor())
+    } else {
+        None
+    }
 }
```

Comment:
This is a bit verbose, I'll try a slicker form afterwards (same repeated two times below in the same file)